### DISCUSSION TITLE
Github Copilot Issue

### DISCUSSION THREAD

**Discussion by AhmadMaherIsmail** [community/community#11070] [Copilot Conversations] [answered]
I was using Copilot for the past week and it has been working perfectly fine. 2 days ago, it started giving me an error that it cannot connect to the server. I uninstalled the extension, logged out of Github on VSCode and redid all the steps

Now, it is giving me another error: read ECONNRESET github copilot

Any solutions regarding that matter?

Thanks

**Comment by davecheney** [comment, marked as answer]
Are you behind a vpn, or, is your internet service provider, employer, or government intercepting your SSL traffic?

Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Reply by luckymore** [reply]
@davecheney 
![image](https://user-images.githubusercontent.com/5390013/164613019-effd12c0-bdb8-4bb5-915a-502e7a64dded.png)

**Reply by davecheney** [reply]
Something or someone is intercepting the traffic between your computer and our servers.
